pom.xml

```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>PaperSpec</artifactId>
-  <version>1.2-SNAPSHOT</version>
+  <version>1.3-SNAPSHOT</version>
```

Comment:
Again. You don't have to bump this
```suggestion
  <version>1.2-SNAPSHOT</version>
```

Reply:
Oops somehow I accidently got rid of the revert commit for this mistake..